src/database/migrations/20190914173000-modify-request-table-with-new-columns.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => Promise.all([
+    queryInterface.addColumn(
+      'Requests',
+      'passportNumber', {
+        type: Sequelize.STRING,
+        allowNull: true
+      }
+    ),
+    queryInterface.addColumn(
+      'Requests',
+      'passportName', {
+        type: Sequelize.STRING,
+        allowNull: true
+      }
+    ),
+    queryInterface.addColumn(
```

Comment:
Please why are you adding this to the request table? The lineManager can be gotten from the User's table with his/her id and roleId

Reply:
These changes can about a result of the feature requirements on the PT stories. What this feature does is to save the user data during a trip request which are dynamic. It's kinda complex, I will explain it over the phone.